open-apparel-registry/open-apparel-registry
# Add Link Back to Facility Claims

## Overview

Previously, there was only a link to click back to the Dashboard at the top of an individual "facility claim" page in the admin interface. Adds a link to take you back to the list of facility claims, rather than just the Dashboard link, to reduce the number of clicks required when working through facility claims.

Connects #828 

## Demo

<img width="1140" alt="Screen Shot 2021-04-08 at 1 33 27 PM" src="https://user-images.githubusercontent.com/21046714/114074476-67cb4300-9872-11eb-8355-e542bf931784.png">

## Testing Instructions

* Run `./scripts/server`
* Create a new facility claim
* Login as `c1@example.com`
* Navigate to `http://localhost:6543/dashboard/claims`
* Select a claim to go to the claim details page
     - [ ] There should be a link to the facility claims page
     - [ ] Clicking the link should take you to the facility claims page

## Checklist
